src/test/java/ui/ConfigProperties.java

```diff
@@ -0,0 +1,31 @@
+package ui;
+
+import java.io.FileInputStream;
+import java.io.IOException;
+import java.util.Properties;
+
+public class ConfigProperties {
+    protected static Properties properties;
+    protected static FileInputStream fileInputStream;
+
+    static {
+        try {
+            fileInputStream = new FileInputStream("src/test/resources/config.properties");
```

Comment:
Reference to `src` directory doesn't seem correct. When you package the app, there will be no such a directory.
There're two ways of reading properties:
* from classpath
* from external file, and its name should be provided in arguments, like `java --external-config=C:\Projects\config.properties -jar project.jar`